Automattic/pocket-casts-ios
# Cleanup build warnings

This PR cleans up a series of build warning that are shown by Xcode, so we don't ignore valid warnings :)

## To test

- CI should be green
- Build and run the app locally and see if you only see two warnings
- Smoke test the main app and the watch OS app
- On the iOS app go to Profile -> Settings and check if the icon for Notification (a bell) is shown
- On the watch app go to Now Playing and if you have more that 9 items on the UpNext queue you should the 9+ icon on the bottom right


## Checklist
